addon.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="script.module.inputstreamhelper" version="0.3.5" name="InputStream Helper" provider-name="emilsvennesson">
+<addon id="script.module.inputstreamhelper" version="0.4.0" name="InputStream Helper" provider-name="emilsvennesson">
```

Comment:
You should definitely add yourself in ```provider-name```. :-)

Reply:
I will leave that up to you. I am not going to do self-promotion :-)